contents/staticAnalysis/ESLint.md

```diff
@@ -75,6 +76,20 @@ ESLint can be easily integrated with your build tool of choice. Here are some [p
 ### Continuous Integration
 ESLint can work with:
 * [Travis](https://medium.com/jason-feng/travis-ci-guide-bdc03b3dbc9d)
+* AppVeyor
+* CircleCI
+
+Simply add the eslint command as your pretest/test script in `package.json`.
+```json
+{
+    ...
+    "scripts": {
+        "pretest": "./node_modules/.bin/eslint ."
+    }
+    ...

```

Comment:
![image](https://user-images.githubusercontent.com/12574756/37016465-4c73f67e-2147-11e8-966a-5b627506bcc9.png)

Those highlighted red dots don't look good. Maybe deleting the `json` part after ``` can remove the highlight?